Formula/grass.rb

```diff
@@ -0,0 +1,352 @@
+      "--with-freetype-includes=#{Formula["freetype"].opt_include}/freetype2",
+      "--with-freetype-libs=#{Formula["freetype"].opt_lib}",
+      "--with-proj-includes=#{Formula["proj"].opt_include}",
+      "--with-proj-libs=#{Formula["proj"].opt_lib}",
+      "--with-proj-share=#{Formula["proj"].opt_share}/proj",
+      "--with-tiff",
+      "--with-pdal=#{Formula["pdal"].opt_bin}/pdal-config",
+      "--with-tiff-includes=#{Formula["libtiff"].opt_include}",
+      "--with-tiff-libs=#{Formula["libtiff"].opt_lib}",
+      "--with-png",
+      "--with-png-includes=#{Formula["libpng"].opt_include}",
+      "--with-png-libs=#{Formula["libpng"].opt_lib}",
+      "--with-regex",
+      # "--with-regex-includes=#{Formula["regex-opt"].opt_lib}",
+      # "--with-regex-libs=#{Formula["regex-opt"].opt_lib}",
+      "--with-fftw",
+      "--with-fftw-includes=#{Formula["fftw"].opt_include}",
+      "--with-fftw-libs=#{Formula["fftw"].opt_lib}",
+      "--with-sqlite",
+      "--with-sqlite-includes=#{Formula["sqlite"].opt_include}",
+      "--with-sqlite-libs=#{Formula["sqlite"].opt_lib}",
+      "--with-mysql",
+      "--with-mysql-includes=#{Formula["mysql"].opt_include}/mysql",
+      "--with-mysql-libs=#{Formula["mysql"].opt_lib}",
+      "--with-postgres",
+      "--with-postgres-includes=#{Formula["postgresql"].opt_include}",
+      "--with-postgres-libs=#{Formula["postgresql"].opt_lib}",
+      # Error: liblas has been disabled because it is not supported upstream!
+      # "--with-liblas",
+      # "--with-liblas-includes=#{Formula["liblas"].opt_include}",
+      # "--with-liblas-libs=#{Formula["liblas"].opt_lib}",
+      "--with-opengl=macosx",
+      "--with-opencl",
+      "--with-openmp",
+      "--with-openmp-includes=#{Formula["libomp"].opt_include}",
+      "--with-openmp-libs=#{Formula["gcc"].opt_lib}/gcc/current",
+      "--enable-macosx-app",
+    ]
+
+    flags << "--with-pthread"
+    flags << "--with-pthread-includes=#{Formula["boost"].opt_include}/boost/thread"
+    flags << "--with-pthread-libs=#{Formula["boost"].opt_lib}"
+
+    sdk_frameworkspath = "#{MacOS.sdk_path}/System/Library/Frameworks"
+
+    flags << "--with-macosx-sdk=#{MacOS.sdk_path}"
+    flags << "--with-opengl-includes=#{sdk_frameworkspath}/OpenGL.framework/Headers"
+    flags << "--with-opengl-framework=#{sdk_frameworkspath}/OpenGL.framework"
+    flags << "--with-opencl-includes=#{sdk_frameworkspath}/OpenCL.framework/Versions/Current/Headers"
+    flags << "--with-opencl-libs=#{sdk_frameworkspath}/OpenCL.framework/Versions/Current/Headers"
+
+    resource("python-six").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+
+    system "./configure", "--prefix=#{prefix}", *flags
+    # system "./configure", "--prefix=#{env['HOME']}/Applications", *flags
+
+    system "sed", "-ibak", "s|/Library|~/Library|g", "macosx/Makefile"
+
+    system "make", "-j", Hardware::CPU.cores
+    system "make", "-j", Hardware::CPU.cores, "install"
```

Comment:
```suggestion
    system "make"
    system "make", "install"
```
`brew` takes care of `-j`.